x-pack/plugins/security_solution/scripts/assistant/attack_discovery/index.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { run } from '@kbn/dev-cli-runner';
+import { HORIZONTAL_LINE } from '../../endpoint/common/constants';
+import { createEsClient, createKbnClient } from '../../endpoint/common/stack_services';
+import { createToolingLogger } from '../../../common/endpoint/data_loaders/utils';
+import { loadAttackDiscoveryData } from './load';
+
+export const cli = () => {
+  run(
+    async (cliContext) => {
+      createToolingLogger.setDefaultLogLevelFromCliFlags(cliContext.flags);
+
+      const log = cliContext.log;
+      const kbnClient = createKbnClient({
+        log,
+        url: cliContext.flags.kibana as string,
+        username: cliContext.flags.username as string,
```

Comment:
nit: String casting doesn't look correct here since it doesn't perform any runtime validation. Flags are parsed in a generic way and it's supposed the handler function performs proper validation and throws an error when a flag doesn't have an expected type. It should be straightforward for `username` and `password` but `kibana` or `elasticsearch` are kind of unclear flags. Ideally we need something like `yargs` provides for input parameter.